test/RbacTest.php

```diff
@@ -121,7 +121,14 @@ public function testHasRole()
         // check that 'snafu' role and $snafu are different
         $this->assertNotEquals($snafu, $this->rbac->getRole('snafu'));
         $this->assertTrue($this->rbac->hasRole('snafu'));
-        $this->assertFalse($this->rbac->hasRole($snafu));
+        $this->assertTrue($this->rbac->hasRole($snafu));
```

Comment:
This is behaviour change, when we provide RoleInstance we expect that Rbac contains the same instance, so we can't compare just names, what I mean:
```php
$role1 = new Role('role');
$role2 = new Role('role');

$role1 !== $role2;
```

Reply:
Great catch!  Fixed this one.